crs/v1alpha2/controlplane_ubuntu.yaml

```diff
@@ -42,22 +42,30 @@ spec:
     - mv /tmp/akeys /home/ubuntu/.ssh/authorized_keys
     - chown ubuntu:ubuntu /home/ubuntu/.ssh/authorized_keys
     - apt update -y
+    - netplan apply
     - >-
-      apt install net-tools gcc linux-headers-$(uname -r) keepalived
+      apt install net-tools gcc linux-headers-$(uname -r) bridge-utils
```

Comment:
I do not think we need these.
gcc
linux-headers-$(uname -r)

Reply:
This PR is not adding those packages so I have not checked whether we can live without those

Reply:
I agree.